src/main/java/frc/robot/commands/Mobility.java

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import edu.wpi.first.wpilibj2.command.RunCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.subsystems.SwerveSubsystem;
+
+
+
+public class Mobility extends CommandBase {
+  /** Creates a new ScoreMobility. */
+  private final SequentialCommandGroup mCommands;
+
+  public Mobility(SwerveSubsystem swerve) {
+    addRequirements(swerve);
+    mCommands = new SequentialCommandGroup(
+        new RunCommand(() -> swerve.setMotors(-1, 0, 0)).withTimeout(4.2),
```

Comment:
how many meters we run backward in 4.2 seconds? Reconsider this logic. This is hard-coded value brittle...

Reply:
We should employ sensors to deal with 'unhappy' cases...

Reply:
The reply has not addressed the "how many meters we run backward in 4.2 seconds" question. During girl gen comps we added sand to the robot making it heavier. Is the heavier robot or different static frictions at different fields will lead ad-hoc timeout change at comps or face potential issues? A fusion of the LIDAR measurements and wheel's encoder readings will lead to more reliable localization and path following.

Reply:
The point was that due to the cable bump, "how many meters we run backward in 4.2 seconds" will be different. The point of this autonomous was a solution that did not use vision or any features that weren't completely tested by the time we ran this at Girls Gen. It was meant to be a backup solution to mobility.